### ISSUE TITLE
Mission Statement

### ISSUE THREAD

**Issue by thejoefriel** [yalla-coop/MASTER-REFERENCE#2]
**In this issue we need to agree the Yalla Cooperative mission statement**

Yalla Cooperative has two key goals: 
1. Drive business opportunities into Gaza 
2. Design and build digital solutions for charities, not-for-profits, start ups and purpose-driven businesses looking to deliver positive social impact

Suggested mission statement is:

"To represent and support freelancers within the cooperative, providing business opportunities in Gaza, and with a focus on delivering digital solutions for clients that drive positive social impact."

What do people think?

**Comment by Oliversw** [comment]
I think that sounds good @thejoefriel! I think the last two points are very clear and make sense in the context of a mission statement but I'd be interested to talk a bit about the first point - what exactly do you mean by "To represent and support freelancers within the cooperative"?

**Comment by thejoefriel** [comment]
Yeah let's definitely chat. I wasn't sure whether we need to outline that the cooperative is freelancers within the mission statement or not. No idea!

**Comment by sofer** [comment]
It could be more snappy. It also might be worth starting with the vision, which could be something like "harnessing the collective talent of the people of Gaza to build better software for people everywhere". Or something like that.

**Comment by amusameh** [comment]
This is my honest opinion and thoughts and hopefully everyone will get the right idea that I’m trying to explain here.
1. The way I think of us in Yalla as a team and shared responsibility that feeling made us work extra hours at nights and during weekends to get work delivered in the right time and cover the inaccuracy of our time estimates.
2. As we are talking about the Yalla mission and big goals. I’m quite afraid of converting Yalla to Aid provider for Gaza by providing jobs for Gazan which is a bad thing at least to me. It’s kind and nice to want to help but this is not the right form of help ( we have UNRWA, UNDP, Mercy corps, USAID, CRS followed by a very long list of such shortcuts) and I don’t want Yalla to somehow degrade to join this list.
   The best way to help Gaza is by building as a real business derived by profit and mutual benefit between the parts of this partnership. I would hate to be hired anywhere because I’m only from Gaza and someone felt the sorry and wanted to help. I want to be employed because of my skills and professional work. That being said I think building Yalla marketing strategies on the fact that it employs Gazan people will lead to be part of the current wheel of donations, aid and pity situation which has been happening in Gaza for years.

   Starting to building agencies with partnership with London or western startups. A relation that is a pure business but on equal mutual benefits (both parts needs each other’s not one side relying on the other) and skills where you collaborate with Gazan because of their skills not their situation is What might lead to a real development in Gaza and breaking the current wheel on converting Gazans to aid seekers by rising up to challenge and building real skills that needed everywhere.
3. Breaking the pity wheel and start thinking of Yalla a business will lead to some important questions about the future and ownership of Yalla.
	- Who owns Yalla? Joe? Joe and Simon? Ramy as well? 
	- What’s Gaza size and representation on the management side of Yalla? For example Ramy has been called a co-founder publicly a few times what that entitles him to? … etc etc
	- If we started building a revenue pot and taking margin from projects and that’s eventually build up to Yalla capital then who owns this capital? Members will own that based on a percentage? Equities? And how is that decided for each member? New members vs old ones, co-founders vs not co-founders, Gaza vs Europe?
	- difference in payments between Gaza and UK and how the scale is decided? Is there any reference for that? 

That’s all in my mind for now but could be followed by lots of questions.
Ps: This is a very delicate topic and I hope that you get the right idea of it.
